pkg/lifecycle/lifecycle.go

```diff
@@ -37,13 +36,10 @@ type WalkDirFunc func(fs billy.Filesystem, dirPath string, doFunc filesystem.Rel
 // InitDependencies will check the filesystem, branch version,
 // git status, initialize the Dependencies struct and populate it.
 // If anything fails the operation will be aborted.
-func InitDependencies(rootFs billy.Filesystem, branchVersion string, currentChart string) (*Dependencies, error) {
+func InitDependencies(repoRoot string, rootFs billy.Filesystem, branchVersion string, currentChart string) (*Dependencies, error) {
 	var err error
 
-	workDir, err := os.Getwd()
-	if err != nil {
-		return nil, err
-	}
+	workDir := repoRoot
```

Comment:
Could alternatively pass in just the repoRoot string and construct the billy.Filesystem inside this function to avoid duplicate parameters in function signatures, although I was ensure if the workDir should ALWAYS be the repoRoot in the autobumps/lifecycle code